net.fasterland.converseen.json

```diff
@@ -0,0 +1,57 @@
+{
+    "id": "net.fasterland.converseen",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "converseen",
+    "rename-desktop-file": "converseen.desktop",
+    "rename-appdata-file": "converseen.appdata.xml",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=home",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+          "name": "imagemagick",
```

Comment:
do you really need this?

Reply:
Yes, this whole application is basically a wrapper around `Magick++`.

Reply:
the binaries or the lib? because if it never uses the binaries, they should be in the cleanup.

Reply:
it seems to require `convert`
also need to cleanup  `/share/doc`, `/include`, and maybe the static libs?

Reply:
> the binaries or the lib?
I guess both then. Added the cleanup rule for the rest.